packages/appeals-service-api/src/services/back-office-v2/formatters/utils.js

```diff
@@ -108,20 +109,27 @@ exports.formatAddresses = (addresses) =>
 
 /**
  * @param {Answers} answers
- * @returns {string[]}
+ * @returns {string[]|null}
  */
 exports.howYouNotifiedPeople = (answers) => {
-	let notifiedPeople = [];
```

Comment:
these questions were not used anymore, instead a single checkbox question that stores answers as a string
have mapped these to the data model values here